### PR TITLE
Ampify logo is too large on the KEDA community page

### PR DESCRIPTION
**Pull request by ofekrom**
_Provide a description of what has been changed_

### Checklist

- [ ] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Comment by zroubalik** [comment]
@ofekrom thanks for this, but the png file is marked as confliciting in this PR, could you please fix that?

**Comment by ofekrom** [comment]
@zroubalik Thanks, fixed.

File: config.toml [hunk 1]
```diff
@@ -289,6 +289,10 @@ icon = "fab fa-microsoft"
 url = "https://alibabacloud.com/"
 logo = "alibaba-cloud.png"
 
+[[params.users]]
+url = "https://ampify.io/"
+logo = "ampify.png"
```
Comment (tomkerkhove) [inline config.toml:294]:
```suggestion
logo = "ampify-io.png"
```

**Comment by tomkerkhove** [comment]
Thanks for making the change - Can you elaborate on what this PR is aimed at because the current preview looks the same as the current live version or am I missing something?

**Pull request closed without merging**